### PR TITLE
add shiden collator page color

### PR DESCRIPTION
**Pull request by hoonsubin**
Add Shiden page color and changed the logo image

### CODE REVIEW

File: packages/apps-config/src/ui/colors.ts [hunk 1]
```diff
@@ -247,10 +248,13 @@ export const chainColors: Record<string, string> = [
...
-  ...colors,
-  [chain.toLowerCase()]: color
-}), {});
+].reduce(
+  (colors, [chain, color]): Record<string, string> => ({
+    ...colors,
+    [chain.toLowerCase()]: color
+  }),
+  {}
+);
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:257]: Not sure why these changes slipped in in all 3 places.

Comment (hoonsubin) [reply packages/apps-config/src/ui/colors.ts:257]: running `yarn lint` seems to change it. It could be an issue with my local config.

Comment (hoonsubin) [reply packages/apps-config/src/ui/colors.ts]: Changed the indents

**Review by jacogr — APPROVED** [review]
